### ISSUE TITLE
Can we update Evospace to 0.19, maybe add CI

### ISSUE THREAD

**Issue by wavelet-noise** [doubleaxe/daxfb-calculator#13]
Hello, looks like data from 0.19 is no longer supported, i have added Colors field to recipes, but not sure what else is incompatible

You can use https://github.com/wavelet-noise/evospace-vanilla-gen/tree/main to generate new json's with Generate.py without steam or evospace client

```
static #recipeSchema = _commons.compileJsonSchema({
        type: 'object',
        additionalProperties: false,
        required: ['Name', 'Ticks', 'Input', 'Output'],
        properties: {
            Input: {$ref: '#/$defs/IO'},
            Output: {$ref: '#/$defs/IO'},
            ResourceInput: {$ref: '#/$defs/Item'},
            ResourceOutput: {$ref: '#/$defs/Item'},
            Name: {type: 'string'},
            Ticks: {type: 'number'},
            Tier: {type: 'integer'},
            Loss: {type: 'integer'},
            Scaled: {type: 'boolean'},
            Locked: {type: 'boolean'},
            
            >>>
            Colors: {
                type: 'array',
                items: {
                    type: 'array',
                    items: {type: 'number'},
                    minItems: 4,
                    maxItems: 4
                }
            }
            <<<
        },
```

**Comment by wavelet-noise** [comment]
I managed to run node prepare.mjs locally with some minor changes, but I can't seem to build the entire site. I'm not very familiar with JS =(

**Comment by wavelet-noise** [comment]
Some icons now not 32x32 also, but still 1:1 and power of 2

**Comment by doubleaxe** [comment]
Ugh, you have compatible json export tool :) I thought I should deal with your new lua format, write converter lua mod, etc.. It should be much easier, I'll take a look a bit later.

What do you think about version 18? Should I keep both versions of calculator or just move everything to v19?

**Comment by wavelet-noise** [comment]
Yes, I decided not to use Lua for vanilla, at least for the basic information. I will need an independent JSON generator for the wiki site in the future myself

The changes in the recipes are minor for now, balance adjustments and changes will come later. I think version 0.18 is not needed yet

**Comment by doubleaxe** [comment]
Fixed in v1.0.21
